readme.mdown

```diff
@@ -1,5 +1,7 @@
 # Scrivito Example App JS
 
+> **⚠️ DEPRECATED**: This repository is deprecated and is no longer maintained. Please use the new [Scrivito Portal App](https://github.com/Scrivito/scrivito-portal-app) instead.
+
```

Comment:
* [ ] How about a more encouraging description?
  * "deprecated" can be misunderstood. This repo is still a good historic reference for how things can be done and nothing in it is deprecated, as Scrivito is 100% backwards compatible.
  * Maybe start with the fact that development has moved?
  * We could also list some of the new features of the Portal App so people get curious to have a look. The notice will also stick out more, and push the old README stuff down.
  * Maybe drop the warning icon? I think GitHub already highlights archived repos.
  * Check with Andreas for wording?